src/screens/app/Components/SettingsButtonVertical/SettingsButtonVertical.js

```diff
@@ -0,0 +1,19 @@
+import {IconButton} from 'react-native-paper';
+import React from 'react';
+
+export default class SettingsButtonHorizontal extends React.Component {
+  state = {
+    iconSize: 30,
+    settingsButtonVerticalIcon: 'dots-vertical',
+  };
+  handleSettingsClick = () => {};
+  render() {
+    return (
+      <IconButton
+        icon={this.state.settingsButtonVerticalIcon}
+        size={this.state.iconSize}
+        onPress={this.handleSettingsClick}
```

Comment:
we can just name component <settingsbutton/> and pass a prop to give it an option to be vertical or horizontal

Reply:
great idea. I'll do this!

Reply:
Completed with unit tests!